apps/fluent-tester/src/TestComponents/Switch/Switch.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { SWITCH_TESTPAGE } from './consts';
+import { View, StyleSheet } from 'react-native';
+import { Switch } from '@fluentui-react-native/switch';
+
+const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: 1,
+  },
+  square: {
+    backgroundColor: 'red',
+    width: 100,
+    height: 100,
+  },
+});
+
+const StandardUsage: React.FunctionComponent = () => {
+  return (
+    <View style={styles.container}>
+      <Switch defaultChecked={true} label={'Default Checked True'} />
+      <Switch defaultChecked={false} label={'Default Checked False'} />
+      <Switch defaultChecked={true} label={'Disabled Default Checked True'} disabled />
+      <Switch defaultChecked={false} label={'Disabled Default Checked False'} disabled />
+    </View>
+  );
+};
+
+const OnChangeUsage: React.FunctionComponent = () => {
+  const defaultDisplaySquare = true;
+  const [displaySquare, setDisplaySquare] = React.useState(defaultDisplaySquare);
```

Comment:
Instead of using `deafultDisplaySquare` to set the initial value of `displaySquare`, you can just use `React.useState(true);`

Reply:
I got rid of `defaultDisplaySquare`, using just `true`